saleor/static/populatedb_data.json

```diff
@@ -10236,5 +10299,15 @@
       "collection": 2,
       "channel": 2
     }
+  },
+  {
+    "model": "product.collectionchannellisting",
```

Comment:
Maybe this collection should be available on both channels?

Reply:
Good point, thanks :)